smile_id_core/web_api.py

```diff
@@ -256,7 +323,11 @@ def upload(url, file):
 
 
 class WebApiLive(WebApiTest):
-    SERVER_URL = Servers.LIVE_SERVER_URL
+    def __init__(self):
```

Comment:
Not sure that this would work :slightly_smiling_face:

first - you are not calling `super().__init__()` on the base class to inherit all of the attributes
second - you never set api_version attr on the class, so the check - `self.api_version == 1` will always be false